wherehows-web/app/routes/logout.js

```diff
@@ -1,17 +1,21 @@
 import Ember from 'ember';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import fetch from 'ember-network/fetch';
 
-const {Route, inject: {service}} = Ember;
-
+const { get, Route, inject: { service } } = Ember;
 
 export default Route.extend(AuthenticatedRouteMixin, {
+  /**
+   * @type {Ember.Service}
+   */
   session: service(),
 
   actions: {
+    /**
+     * Post transition, call endpoint then invalidate current session on client on success
+     */
     didTransition() {
-      Ember.$.get('/logout').then(() => {
-      this.get('session').invalidate();
-      });
+      fetch('/logout').then(() => get(this, 'session').invalidate());
```

Comment:
If the logout fails, at this point the user will remain on the page and will probably re-attempt to logout.